HeterogeneousCore/MPICore/scripts/cmsenv_mpirun

```diff
@@ -2,25 +2,37 @@
 
 VERBOSE=false
 RELEASE=false
+MCA_PREFIX=cmsenv
 
 for ARG in "$@"; do
   case "$ARG" in
     -v|--verbose)
       VERBOSE=true
       ;;
-    cmsenv_release)
+    ${MCA_PREFIX}_release)
       RELEASE=true
       ;;
   esac
 done
 
-# additional options oassed to cmsenv_orted
+# full path to cmsenv_orted
+RELEASE_BASE=
+if [ -x $CMSSW_BASE/bin/$SCRAM_ARCH/cmsenv_orted ]; then
+  RELEASE_BASE=$CMSSW_BASE
+elif [ -x $CMSSW_RELEASE_BASE/bin/$SCRAM_ARCH/cmsenv_orted ]; then
+  RELEASE_BASE=$CMSSW_RELEASE_BASE
+else
```

Comment:
I believe it could make sense to check also `$CMSSW_FULL_RELEASE_BASE` for the case that `$CMSSW_RELEASE_BASE` is a patch release. (alternative could be to find the location of `cmsenv_orted` from $PATH, but I'm not sure if that could have some undesired effect)

Reply:
Good point, thanks.